source/daemon/src/daemon.cpp

```diff
@@ -6,10 +6,9 @@
 #include <daemon.h>
 #include <gpio_pin_definitions.h>
 #include <gpio_mapping.h>
-//#include <calib_struct.h>
 #include <ublox_messages.h>
 #include <tcpmessage_keys.h>
-#include <iomanip>      // std::setfill, std::setw
```

Comment:
Why delete this comment? I find it quite useful to see for which functions a header is included...